resources/assets/components/CampaignSingle/index.js

```diff
@@ -55,32 +56,21 @@ class CampaignSingle extends React.Component {
     });
   }
 
-  // Make API call to GET /posts to get posts by filtered status.
-  getPostsByStatus(status, campaignId) {
+  // Make API call to GET /posts to get posts by filtered status and/or tag(s).
+  getPostsByFilter(campaignId, status, activeTags) {
```

Comment:
Could we possibly update this to take in an object and then use the properties in the object to send in the request? If we ever wanted to filter by users, runs, etc we would have to keep adding params here. So I'm wondering if we use the filter values set by the user to create the `filter` array we send in the request.